dockerfiles/vcpkg/ubuntu-22.04.dockerfile

```diff
@@ -1,9 +1,14 @@
 ARG VCPKG_REF=latest
-FROM hpccbuilds/vcpkg-ubuntu-22.04:$VCPKG_REF
+ARG DOCKER_NAMESPACE=hpccbuilds
+FROM ${DOCKER_NAMESPACE}/vcpkg-ubuntu-22.04:${VCPKG_REF}
```

Comment:
This will change shortly (you may need to change it depending on which PR get pulled first)